src/main/java/org/fundacionjala/coding/william/CamelCase.java

```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.coding.william;
+
+import java.util.StringJoiner;
+
+/**
+ * Class that converts the first letter of a word into a capital letter.
+ */
+public class CamelCase {
+
+    /**
+     * Method that converts the first letter of a word into a capital letter.
+     *
+     * @param word receives a string.
+     * @return returns a string.
+     */
+    public String camelCase(final String word) {
+        String[] array = word.split(" ");
+        StringJoiner result = new StringJoiner("");
+        String[] auxiliary;
+        for (int i = 0; i < array.length; i++) {
+            auxiliary = array[i].split("");
+            auxiliary[0] = auxiliary[0].toUpperCase();
```

Comment:
To find the first letter is not efficient this way
You can handle as String.
Please change it.